spec/unit/parse_yaml_spec.rb

```diff
@@ -7,10 +7,6 @@
       expect(process(nil).body).to be_nil
     end
 
-    it "returns false for empty body" do
-      expect(process('').body).to be false
-    end
-
```

Comment:
`YAML.safe_load` (which is part of Psych, which is included in Ruby from 2.x onwards) returns `nil`. 

But the `safe_yaml` gem returns `false` still, unfortunately. Rather than have a conditional test, I elected to remove this. The better solution would be to stop supporting Ruby 1.x, which would vastly improve this code, and makes sense given that [1.9 reached end of life more than 2 years ago](https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/).